data/kobold_press/document.json

```diff
@@ -0,0 +1,245 @@
+            "Beyond Damage Dice © 2016 Open Design; Author: James J. Haeck.",
+            "Birds of a Feather Author Kelly Pawlik. © 2019 Open Design LLC.",
+            "Black Sarcophagus Author: Chris Lockey. © 2018 Open Design LLC.",
+            "Blood Vaults of Sister Alkava. © 2016 Open Design. Author: Bill Slavicsek.",
+            "Book of Lairs for Fifth Edition. Copyright 2016, Open Design; Authors Robert Adducci, Wolfgang Baur, Enrique Bertran, Brian Engard, Jeff Grubb, James J. Haeck, Shawn Merwin, Marc Radle, Jon Sawatsky, Mike Shea, Mike Welham, and Steve Winter.",
+            "Casting the Longest Shadow. © 2020 Open Design LLC. Author: Benjamin L Eastman.",
+            "Cat and Mouse © 2015 Open Design; Authors: Richard Pett with Greg Marks.",
+            "Courts of the Shadow Fey © 2019 Open Design LLC; Authors: Wolfgang Baur & Dan Dillon.",
+            "Creature Codex. © 2018 Open Design LLC; Authors Wolfgang Baur, Dan Dillon, Richard Green, James Haeck, Chris Harris, Jeremy Hochhalter, James Introcaso, Chris Lockey, Shawn Merwin, and Jon Sawatsky.",
+            "Creature Codex Lairs. © 2018 Open Design LLC; Author Shawn Merwin.",
+            "Dark Aerie. ©2019 Open Design LLC. Author Mike Welham.",
+            "Death of a Mage ©2020 Open Design LLC. Author R P Davis.",
+            "Deep Magic © 2020 Open Design LLC; Authors: Wolfgang Baur, Richard Green, Jeff Lee, Christopher Lockey, Kelly Pawlik, Mike Welham.",
+            "Deep Magic for 5th Edition © 2020 Open Design LLC; Authors: Dan Dillon, Chris Harris, and Jeff Lee.",
+            "Deep Magic: Alkemancy © 2019 Open Design LLC; Author: Phillip Larwood.",
+            "Deep Magic: Angelic Seals and Wards © 2016 Open Design; Author: Dan Dillon.",
+            "Deep Magic: Battle Magic © 2016 Open Design; Author: Greg Marks.",
+            "Deep Magic: Blood and Doom © 2017 Open Design; Author: Chris Harris.",
+            "Deep Magic: Chaos Magic © 2016 Open Design; Author: Greg Marks.",
+            "Deep Magic: Clockwork © 2016 Open Design; Author: Scott Carter.",
+            "Deep Magic: Combat Divination © 2019 Open Design LLC; Author: Matt Corley.",
+            "Deep Magic: Dragon Magic © 2017 Open Design; Author: Shawn Merwin.",
+            "Deep Magic: Elemental Magic © 2017 Open Design; Author: Dan Dillon.",
+            "Deep Magic: Elven High Magic © 2016 Open Design; Author: Greg Marks.",
+            "Deep Magic: Hieroglyph Magic © 2018 Open Design LLC; Author: Michael Ohl.",
+            "Deep Magic: Illumination Magic © 2016 Open Design; Author: Greg Marks..",
+            "Deep Magic: Ley Line Magic © 2016 Open Design; Author: Dan Dillon.",
+            "Deep Magic: Mythos Magic © 2018 Open Design LLC; Author: Christopher Lockey.",
+            "Deep Magic: Ring Magic © 2016 Open Design; Author: Dan Dillon.",
+            "Deep Magic: Runes © 2016 Open Design; Author: Chris Harris.",
```

Comment:
To me, at least, that seems like a very reasonable way to split them up. And it also seems like a decision we could revisit if we change our minds and want to get more fine-grained. Shouldn't break anything for items to change their source docs.

I may try and go through and clean up the license sections so they're only listing the actual sources referenced, if you don't mind? As-is they're all very very long, and reference the same list of Kobold Press publications.

Reply:
Feel free. I didn't know if any spells used more outside of the sources they are tagged with, e.g. WotC SRD, so I included everything just in case. I did leave the original source field on each spell in case there needs to be some reshuffling, since most appear in more than one book.

Reply:
Great thinking.

I'm gonna assume for now that the sources they're tagged with are accurate, and as long as we credit at least one source we should be good. Will pull this down over the weekend, cut those down, then get it all merged in! Thank you for all this.